lib/reporters/html/reporterTemplate.html

```diff
@@ -0,0 +1,204 @@
+                            </th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        <tr>
+                            <th>Property</th>
+                            <th>Iterations</th>
+                            <th>Requests</th>
+                            <th>Test scripts</th>
+                            <th>Pre-request scripts</th>
+                            <th>Assertions</th>
+                        </tr>
+
+                        <tr>
+                            <th>Executed</th>
+                            {{#with executed}}
+                                <td>{{iterations}}</td>
+                                <td>{{requests}}</td>
+                                <td>{{testScripts}}</td>
+                                <td>{{preRequestScripts}}</td>
+                                <td>{{assertions}}</td>
+                            {{/with}}
+                        </tr>
+
+                        <tr>
+                            <th>Failed</th>
+                            {{#with failed}}
+                                <td>{{iterations}}</td>
+                                <td>{{requests}}</td>
+                                <td>{{testScripts}}</td>
+                                <td>{{preRequestScripts}}</td>
+                                <td>{{assertions}}</td>
+                            {{/with}}
+                        </tr>
+                    </tbody>
+
+                    <tfoot></tfoot>
+                </table>
+            {{/with}}
+
+            <br>
+
+            {{#with statistics}}
+                <table>
+                    <thead>
+                        <tr>
+                            <th colspan="3">
+                                <h1>Run Statistics</h1>
+                            </th>
+                        </tr>
+                        <tr>
+                            <th>Total run duration</th>
+                            <th>Bytes transferred out</th>
+                            <th>Bytes transferred in</th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        <tr>
+                            <td>{{runDuration}}</td>
+                            <td>{{bytes.out}}</td>
+                            <td>{{bytes.in}}</td>
+                        </tr>
+                    </tbody>
+                </table>
+            {{/statistics}}
+
+            <br>
+
+            {{#with metadata}}
+                <table>
+                    <thead>
+                        <tr>
+                            <th colspan="6">
+                                <h1>Run Metadata</h1>
+                            </th>
+                        </tr>
+
+                        <tr>
+                            <th colspan="2">Environment variables</th>
+                            <th colspan="2">Global variables</th>
```

Comment:
Cannot output variables here. Makes the report too long as variables can be huge. Instead you can simply show count of variables.